agent/handlers/task_server_setup_test.go

```diff
@@ -437,15 +438,42 @@ var (
 			Type: containerType,
 		},
 	}
-	expectedV4TaskResponse = v4.TaskResponse{
-		TaskResponse: &v2.TaskResponse{
+	expectedV4BridgeContainerResponse = v4ContainerResponseFromV2(expectedBridgeContainerResponse, []v4.Network{{
+		Network: tmdsresponse.Network{
+			NetworkMode:   bridgeMode,
+			IPv4Addresses: []string{bridgeIPAddr},
+		},
+		NetworkInterfaceProperties: v4.NetworkInterfaceProperties{
+			AttachmentIndex:          nil,
+			IPV4SubnetCIDRBlock:      "",
+			MACAddress:               "",
+			PrivateDNSName:           "",
+			SubnetGatewayIPV4Address: "",
+		}},
+	})
+)
+
+// Creates a v4 ContainerResponse given a v2 ContainerResponse and v4 networks
+func v4ContainerResponseFromV2(
+	v2ContainerResponse v2.ContainerResponse, networks []v4.Network) v4.ContainerResponse {
+	v2ContainerResponse.Networks = nil
+	return v4.ContainerResponse{
+		ContainerResponse: &v2ContainerResponse,
+		Networks:          networks,
+	}
+}
+
+// Returns a standard v4 task response. This getter fucntion protects against tests mutating
```

Comment:
typo(non-blocking): fucntion -> function

Reply:
Thanks! Fixed it.